front/app/containers/Admin/pagesAndMenu/EditHomepage/HeroBanner/index.tsx

```diff
@@ -145,6 +145,7 @@ const EditHomepageHeroBannerForm = () => {
 
   const handleOnBannerImageRemove = () => {
     handleOnChange('header_bg', null);
+    handleOnOverlayChange(null, null);
```

Comment:
If you remove an image now, the overlay will also be turned off. Thought that was desirable behaviour while I was trying it.